### DISCUSSION TITLE
invitation screen changes in 2.3.0

### DISCUSSION THREAD

**Discussion by proxy318** [Budibase/budibase#9637] [Help] [answered]
I have Google SSO enabled, and I believe that previously, when an invitation was sent to a new user, the invitation screen had a "sign in with google" button. That is now missing, and new users have to create a password to sign in. Once they've signed in, they can then sign out and sign back in with google, but this seems unnecessary. Is it possible to get the sign in with google button back on the new account invitation screen?

**Comment by melohagan** [comment, marked as answer]
Hey @proxy318 

In previous versions the invited user has always had to enter in a password to sign up.

In **2.2** the user would be prompted to sign-in after signing up:
![2 2_login](https://user-images.githubusercontent.com/101575380/222137334-81fd6048-be33-4abb-8f2c-d24daeea5162.png)

![2 2_signup](https://user-images.githubusercontent.com/101575380/222136745-cc8def28-b539-4ac3-8b4a-0010e03f7593.png)

Whereas in **2.3** the login occurred automatically after signup. However if that user logs out, and Google SSO is configured correctly, this is what they should see:
![Screenshot 2023-03-01 at 12 17 50](https://user-images.githubusercontent.com/101575380/222137101-9e359a95-c53a-42e8-9d16-ff260a4df784.png)

I believe the upcoming release will allow you to prevent users from using a password, i.e. force them to use SSO: https://github.com/Budibase/budibase/issues/3754

**Comment by proxy318** [comment]
OK, thanks. Just want to see if I can give my users the easiest experience
possible.

Art
